package.json

```diff
@@ -80,21 +80,21 @@
     "guidemaker-ember-template": "^1.3.0",
     "loader.js": "^4.7.0",
     "lodash": "^4.17.21",
-    "markdown-link-extractor": "^1.2.2",
+    "markdown-link-extractor": "^1.2.7",
     "mocha": "^7.0.0",
     "node-fetch": "^2.6.0",
     "npm-run-all": "^4.1.5",
     "prember": "^1.0.3",
     "qunit-dom": "^1.0.0",
-    "remark-cli": "^8.0.0",
-    "remark-lint": "^7.0.0",
-    "remark-preset-lint-consistent": "^3.0.0",
-    "remark-preset-lint-recommended": "^4.0.0",
+    "remark-cli": "^9.0.0",
+    "remark-lint": "^8.0.0",
+    "remark-preset-lint-consistent": "^4.0.0",
+    "remark-preset-lint-recommended": "^5.0.0",
```

Comment:
TODO:

Updating `remark`-related packages slowed down `test:node` script to the point that CI timed out. Investigate why in a separate issue.

Reply:
Update: The problem seems to occur when we update `markdown-link-extractor`.